lib/config/presets/internal/monorepo.ts

```diff
@@ -244,13 +245,13 @@ export const presets: Record<string, Preset> = {};
 
 for (const [name, value] of Object.entries(repoGroups)) {
   presets[name] = {
-    description: `${name} monorepo`,
+    description: gt.gettext(`${name} monorepo`),
```

Comment:
I'm not sure if this works as expected. the passed string is expanded at runtime before passed to gettext function.

Reply:
I doubt whether this string are translatable. "monorepo" is a technical slang, should not be translated. I will remove the function call here.